hotel-management/src/components/Form/FormInput.tsx

```diff
@@ -0,0 +1,25 @@
+import './index.css';
+
+type FormInputProps = {
+  borderRadius: string;
```

Comment:
There should be strict typescript for this prop, if not like this I can pass any string into this prop